package.json

```diff
@@ -41,6 +41,7 @@
     "@typescript-eslint/eslint-plugin": "5.51.0",
     "@typescript-eslint/parser": "5.51.0",
     "eslint": "8.33.0",
```

Comment:
Please revert this change. This is the wrong package.json. You need to add it to projects/eslint-config one.
On top of that, you need to add this to both dependencies and devDependencies.

Reply:
Done